src/server/api/routes/vessel.router.js

```diff
@@ -65,4 +67,15 @@ router.patch(
   }
 );
 
+//ENDPOINT: /api/vessels/:vessel_id/vessel_reports/
+router.get(
+  "/:id/vessel_reports",
```

Comment:
Make sure to add authentication back in for all of your routes. I just tested adding it in for one route and it worked fine. If you had problems, maybe you had forgot to get a new token? It runs out every day.

Reply:
And change the path here to `/:id/vessel-reports`

Reply:
Added authentication back to the all routes and change path from /:id/vessel_reports to /:id/vessel-reports